concourse/pipelines/create-deployer.yml

```diff
@@ -256,7 +258,7 @@ jobs:
             . vpc-terraform-outputs/tfvars.sh
             openssl req -x509 -newkey rsa:2048 -keyout concourse.key \
               -out concourse.crt -days 365 -nodes -subj \
-              "/C=UK/ST=London/L=London/O=GDS/CN=deployer.{{deploy_env}}.${TF_VAR_system_dns_zone_name}"
+              "/C=UK/ST=London/L=London/O=GDS/CN=deployer.${ENV_DNS_ZONE_NAME}"
```

Comment:
With this change, we no longer need the `vpc-terraform-outputs` in this task. Would be good to remove them.